themes/tdr/login/messages/messages_en.properties

```diff
@@ -23,7 +23,11 @@ loggedInTitle=The National Archives Transfer Digital Records
 mainHeader=Transfer Digital Records
 passwordNew=New password
 passwordReset=Password reset
-passwordRestrictions=Password must be at least 8 characters
+passwordInfo=Enter a new password below.
+passwordRestrictionsInfo=Your password must have:
```

Comment:
Maybe rename these a bit to make it clear what they are, it'll just make it clearer in the template what the messages are, something like:

- `passwordRestrictionCharacters`
- `passwordRestrictionCapitalLetter`
- `passwordRestrictionNumber`